core/src/main/java/de/danoeh/antennapod/core/preferences/PlaybackPreferences.java

```diff
@@ -43,6 +43,9 @@
 	/** True if last played media was a video. */
 	public static final String PREF_CURRENT_EPISODE_IS_VIDEO = "de.danoeh.antennapod.preferences.lastIsVideo";
 
+	/** Value of PREF_QUEUE_ADD_TO_FRONT if no media is playing. */
+	public static final String PREF_QUEUE_ADD_TO_FRONT = "prefQueueAddToFront";
```

Comment:
I'm actually not sure this belongs in 'PlaybackPreferences.java'. The preference isn't related to playback.  PreferenceController.java, which I referenced earlier, is probably more appropriate.  If you put it there, this format is fine.

Move this preference there.